README.md

```diff
@@ -301,6 +301,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     A Greek community of people dedicated to promoting women's engagement with technology & science. Its main activities consist of free workshops, meetups open to anyone interested, as well as partnerships with other tech communities and local chapters of Thessaloniki.
 
+  - [Watermark Conference for Women](http://www.watermarkconferenceforwomen.org/conference/)
+
+    Watermark’s Conference for Women offers the community and connection, the information and inspiration, the motivation and momentum to help you discover what you want—and go get it! The conference features nationally recognized speakers who share their wisdom and expertise on a wide range of personal and professional development topics, helping you find clarity on your goals and what you need to accomplish them.
```

Comment:
Not bad. However we don't represent the organization and can't talk for them or market them. Rather we just supply information about them, so we'll need a rewrite with the same amount of information as before.